.github/workflows/gh-actions-pr.yml

```diff
@@ -6,183 +6,367 @@ jobs:
+          #  OpenGL_GL_PREFERENCE: 'LEGACY'
+          #  ENABLE_PIE: 'ON'
+          #  allow_failure: false
+          #- FROM:     'fedora:38'
+          #  COMPILER: 'clang'
+          #  OpenGL_GL_PREFERENCE: 'GLVND'
+          #  ENABLE_PIE: 'ON'
+          #  allow_failure: false
           - FROM:     'fedora:37'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'fedora:37'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'GLVND'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'fedora:36'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'fedora:36'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'GLVND'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'fedora:35'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'fedora:35'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'GLVND'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
+          - FROM:     'rockylinux:9.1'
+            COMPILER: 'clang'
+            OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
+          - FROM:     'rockylinux:9.1'
+            COMPILER: 'clang'
+            OpenGL_GL_PREFERENCE: 'GLVND'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'rockylinux:9.0'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'rockylinux:9.0'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'GLVND'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
+          - FROM:     'rockylinux:8.7'
+            COMPILER: 'clang'
+            OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
+          - FROM:     'rockylinux:8.7'
+            COMPILER: 'clang'
+            OpenGL_GL_PREFERENCE: 'GLVND'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'rockylinux:8.6'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'rockylinux:8.6'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'GLVND'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'rockylinux:8.5'
             COMPILER: 'clang'
             OpenGL_GL_PREFERENCE: 'LEGACY'
+            ENABLE_PIE: 'ON'
+            allow_failure: false
           - FROM:     'rockylinux:8.5'
```

Comment:
Also, this looks way more than just updating boost. It's better and easier to submit several smaller PRs than a big one.

Reply:
Normally, I would agree. In this case, I was just trying to hurry up and get everything building again.

Good point that this is about a lot more than updating Boost. I have updated the PR title to reflect that.